File: frontend/src/app/dto/vehicleDto.ts
export class VehicleDto {
    id: number;
    positionLat: number;
    positionLng: number;
    notification: string;

    constructor(id: number, positionLat: number, positionLng: number, notification: string) {
        this.id = id;
        this.positionLat = positionLat;
        this.positionLng = positionLng;
        this.notification = notification;
    }
 }

File: backend/src/main/java/com/vma/backend/dto/VehicleDto.java
package com.vma.backend.dto;

public class VehicleDto {

	private Integer id;

	private Double positionLat;

	private Double positionLng;

	private String notification;

	public Integer getId() {
		return id;
	}

	public void setId(Integer id) {
		this.id = id;
	}

	public Double getPositionLat() {
		return positionLat;
	}

	public void setPositionLat(Double positionLat) {
		this.positionLat = positionLat;
	}

	public Double getPositionLng() {
		return positionLng;
	}

	public void setPositionLng(Double positionLng) {
		this.positionLng = positionLng;
	}

	public String getNotification() {
		return notification;
	}

	public void setNotification(String notification) {
		this.notification = notification;
	}
}
